### DISCUSSION TITLE
Installation errors

### DISCUSSION THREAD

**Discussion by pxrpq** [Security-Onion-Solutions/securityonion#12828] [General] [answered]
I'm trying to install eval to monitor a LAN. 
I've set the required hardware requirements.
I have 2 adapters: 1) NAT for management 2) Bridged for monitoring.
Regardless of how I tweak the config, the installation keeps failing.

Most recent latest error message :
Install had a problem. Please see /root/sosetup.log.
A summary of errors can be found in /root/errors.log.

I ran: vim /root/errors.log and:
It says "failed to download metadata for repo 'securityonionsync': cannot download repomd.xml:
cannot download repodata/repomd.xml: All mirrors were tried.

...server returned 000 status code. No connectivity to Github... exiting.

**Comment by oklier** [comment]
Hello,
If using a DNS or URL filer with your firewall, might want to check for blocks and allow the traffic.
Regards,

**Reply by pxrpq** [reply]
I disabled the firewall but same result. Thank you for responding

**Comment by reyesj2** [comment]
Is this a virtual machine? It seems there is an issue with your network configuration. Can you ping any external website from the VM like www.google.com?

If you're having problems with network installs you can always use an airgap install and you won't need to connect to any internet resource for your install. https://docs.securityonion.net/en/2.4/airgap.html#airgap

**Reply by pxrpq** [reply]
Yes,  it is a VM. Same result for both virtual box and VMware. 

I wasn't able to ping google. 

I've never encountered this issue before. How do I resolve it?

I appreciate your help

**Reply by reyesj2** [reply]
I would recommend checking out our YouTube channel. There is a great video that will get you through the entire Security Onion installation using VMware PART 1 https://www.youtube.com/watch?v=KMH7cII3mpA PART 2 https://www.youtube.com/watch?v=DQkCg5P9FSw

**Reply by pxrpq** [reply]
I watched multiple times and followed the steps

**Reply by reyesj2** [reply, marked as answer]
If you're having trouble installing you can select the 'airgap' option which will only use the ISO to install all of the Security Onion resources. No need for any internet.
